agenta-cli/agenta/sdk/types.py

```diff
@@ -231,3 +234,14 @@ class ConfigurationResponse(LifecyclesResponse):
 
 class DeploymentResponse(LifecyclesResponse):
     pass
+
+
+class Prompt(BaseModel):
```

Comment:
Why is this part of the documentation PR?

Reply:
Isn't this already part of the release?

Reply:
> Isn't this already part of the release?

Yes, it is. 

> Why is this part of the documentation PR?

It's a last minute change after realizing that the `Prompt` model wasn't included after the `Prompts Client` sanity check / QA. 😞